tests/test_cinoh_city_council.py

```
@@ -0,0 +1,75 @@
+import json
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import CITY_COUNCIL, COMMITTEE
```

Ruff

`city_scrapers_core.constants.COMMITTEE` imported but unused

Remove unused import: `city_scrapers_core.constants.COMMITTEE`

(F401)